common/changes/@snowplow/browser-tracker-core/bt-typedefs_2024-01-30-01-31.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@snowplow/browser-tracker-core",
+      "comment": "",
```

Comment:
Is it a problem that there is no comment in these changes?

Reply:
This will be used in the generated changelog _(which I think we don't use anyway)_
I don't think the empty entry will make a difference _(think :P )_